docs/README.md

```diff
@@ -1,6 +1,8 @@
 # How to run the doc site locally
 
-The `docs` folder contains content published to [Salesforce Extensions for Visual Studio Code](https://developer.salesforce.com/tools/vscode)
+- The `docs` folder contains content published to [Salesforce Extensions for Visual Studio Code](https://developer.salesforce.com/tools/vscode).
+- To preview the rendered doc site from a Github Pull request click "Details" against the "netlify/salesforcedx-vscode/deploy-preview — Deploy Preview ready!" check.
```

Comment:
where does this live?  I don't see that check. 
![Screen Shot 2022-03-22 at 1 16 17 PM](https://user-images.githubusercontent.com/76090802/159548720-c66d23d7-b2b2-41d3-b535-403f74cacf5e.png)

Reply:
That is weird. I see it right in the center among the checks 

![image](https://user-images.githubusercontent.com/1509984/159549820-cd575009-7ab4-4f45-8a26-12caef70ff1d.png)